### PR TITLE
Helm Chart: Release v0.5.2

### PR DESCRIPTION
**Pull request by stefanprodan**

### CODE REVIEW

File: chart/flux/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+
+ - Updated Flux to `v1.8.2` and Helm operator to `v0.5.2`
+   [weaveworks/flux#1612](https://github.com/weaveworks/flux/pull/1612)
+ - Parameterized the memcached image repo
+   [weaveworks/flux#1592](https://github.com/weaveworks/flux/pull/1592)
+ - Allow existing service account to be provided on helm install
+   [weaveworks/flux#1589](https://github.com/weaveworks/flux/pull/1589)
+ - Make SSH known hosts volume optional
+   [weaveworks/flux#1544](https://github.com/weaveworks/flux/pull/1544)
+
```
Comment (hiddeco) [inline chart/flux/CHANGELOG.md]: `### Thanks`?

I didn't do this on the initial setup of the `CHANGELOG.md` as it was too much of a burden. But now that we are up to date I think it is an easy 2 cents we can give back to our contributors :-).

File: chart/flux/Chart.yaml [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 sources:
 - https://github.com/weaveworks/flux
 maintainers:
   - name: stefanprodan
     email: stefan@weave.works
 engine: gotpl
-icon: https://landscape.cncf.io/logos/weave-flux.svg
+icon: https://raw.githubusercontent.com/weaveworks/flux/master/site/images/weave-flux.png
+keywords:
+  - gitops
```
Comment (hiddeco) [inline chart/flux/Chart.yaml]: I assume Helm has no problems parsing this but there is a new line missing.

**Comment by squaremo** [comment]
> the version was bumped in several PRs but 0.5.2 is next, check https://github.com/weaveworks/flux/tree/gh-pages

How confusing, I'm glad someone was paying attention! We should not change the version except when actually releasing it, right? (i.e., those PRs should not have changed it)

**Comment by hiddeco** [comment]
@squaremo this was exactly what I was talking about the other day on Slack. Version number should indeed not be bumped before we are prepping the release.

**Pull request merged**